docs/api.rst

```diff
@@ -100,6 +100,12 @@ Ensembles Module
 .. autofunction:: xclim.ensembles.robustness_coefficient
    :noindex:
 
+.. automodule:: xclim.ensembles._partitioning
+    :noindex:
+
+.. autofunction:: xclim.ensembles.hawkins_sutton
+    :noindex:
+
```

Comment:
Should be also document the functions in `ensembles._filters` ?

Reply:
I wasn't sure they were ready for prime-time. The tests for those are fairly basic.